### ISSUE TITLE
Complete rewrite of codebase

### ISSUE THREAD

**Issue by MidSpike** [MidSpike/iris-utilities#108]
# About
It has come to my attention that I.R.I.S. is in need of a complete rewrite from the ground up.

# Reasoning
Several factors have led me to this conclusion, such as:
- Discord.js v13 coming out soon
- Discord.js v13 dropping voice support
- [Message Content Access Deprecation for Verified Bots](https://support-dev.discord.com/hc/en-us/articles/4404772028055)
- Discord forcing slash commands onto verified bots (see above)
- needing an actual database (I have chosen MongoDB for the rewrite)
- overall poor code quality in retrospect

# Information
Currently I am undertaking this task in a private repository (and private discord bot).  
I will make the source-code available once it is ready to run under the I.R.I.S. Utilities discord bot.

---

# Edit: [Status Update](https://github.com/MidSpike/iris-utilities/issues/108#issuecomment-1088000200)

**Comment by DJHARDY2K21** [comment]
This update will be great! Can't wait too see it!

**Comment by MidSpike** [comment]
# Long-Overdue Status Report

I want to start this off being brutally honest.

It's become increasingly difficult to keep up with the changes that both Discord and Discord.js are requiring from developers.

It seems like everytime that I am back in the groove of making discord bots, Discord decides to make another stupid decision with an extraordinarily small window of time to adapt to.

Adding on to that, all of the annoying changes being made by Discord.js, I'm looking at you *v13 dropping voice support b/c why tf not*.

And with Discord.js v14 just around the corner **omfg another major version bump now with __even more__ breaking changes**. 🤦

Anyways, rant over.

I'm still continuing work on the replacement codebase for I.R.I.S. Utilities.
However work is being conducted at an ever-growing-ly slower pace.

The essential (most used) features are there, like playing music from YouTube, basic media controls, ect.

However, many of the *under-appreciated* (barely-used) features are still pending development.

---

# TL:DR;

I'm still working on the new codebase and am suffering from burnout.
Also Discord.js slow tf down, and keep a consistent api please!

**Comment by MidSpike** [comment]
Rewrite has been merged
